2025-03-31 15:17:11,978 - bluelog - ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask_debugtoolbar\__init__.py", line 222, in dispatch_request
    return view_func(**view_args)
  File "D:\tools\Anaconda\envs\Study\lib\cProfile.py", line 110, in runcall
    return func(*args, **kw)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask_caching\__init__.py", line 426, in decorated_function
    rv = self._call_fn(f, *args, **kwargs)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask_caching\__init__.py", line 185, in _call_fn
    return ensure_sync(fn)(*args, **kwargs)
  File "D:\tools\env\Python\flask\Bluelog\bluelog\blueprints\blog.py", line 21, in index
    return render_template('blog/index.html', pagination=pagination, posts=posts)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: blog/index.html
2025-03-31 15:17:13,036 - bluelog - ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask_debugtoolbar\__init__.py", line 222, in dispatch_request
    return view_func(**view_args)
  File "D:\tools\Anaconda\envs\Study\lib\cProfile.py", line 110, in runcall
    return func(*args, **kw)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask_caching\__init__.py", line 426, in decorated_function
    rv = self._call_fn(f, *args, **kwargs)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask_caching\__init__.py", line 185, in _call_fn
    return ensure_sync(fn)(*args, **kwargs)
  File "D:\tools\env\Python\flask\Bluelog\bluelog\blueprints\blog.py", line 21, in index
    return render_template('blog/index.html', pagination=pagination, posts=posts)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: blog/index.html
2025-03-31 15:17:15,276 - bluelog - ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask_debugtoolbar\__init__.py", line 222, in dispatch_request
    return view_func(**view_args)
  File "D:\tools\Anaconda\envs\Study\lib\cProfile.py", line 110, in runcall
    return func(*args, **kw)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask_caching\__init__.py", line 426, in decorated_function
    rv = self._call_fn(f, *args, **kwargs)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask_caching\__init__.py", line 185, in _call_fn
    return ensure_sync(fn)(*args, **kwargs)
  File "D:\tools\env\Python\flask\Bluelog\bluelog\blueprints\blog.py", line 21, in index
    return render_template('blog/index.html', pagination=pagination, posts=posts)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\tools\Anaconda\envs\Study\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: blog/index.html
